### PR TITLE
integration-tests: increase the revision of the fake update

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: snap/info_snap_yaml.go [hunk 1]
```diff
@@ -70,6 +71,7 @@ func InfoFromSnapYaml(yamlData []byte) (*Info, error) {
 		Name:        y.Name,
 		Developer:   y.Developer,
 		Version:     y.Version,
+		Revision:    y.Revision,
```
Comment (elopio) [inline snap/info_snap_yaml.go]: @chipaca I need you here. I have no idea if this change was in your roadmap, or if it doesn't make sense.

**Comment by mvo5** [comment]
First, let me say a HUGE thank you for making the integration tests green again \o/

I will leave the authoritative answer to @chipaca but I do not think we wan tthe revision in the snap.yaml data structure. I can imagine that we have a aux data structure/file in the fake store that stores information that is not part of the snap.yaml. This could be the developer and the revision for now. Maybe something like  https://github.com/ubuntu-core/snappy/compare/master...mvo5:bugfix/fake-store-revision?expand=1 is all it takes, I will try that.

File: snap/info_snap_yaml.go (hunk 1 above)
Comment (chipaca) [inline snap/info_snap_yaml.go]: Sadly, it doesn't. I mean it *makes sense*, but it's not going to happen. Revision number is external to the thing being revisioned. You want something like `MkStoreYaml` from `snap/removed/removed_test.go`:
```go
func (s *removedSuite) MkStoreYaml(c *check.C, pkgType snap.Type) {
        // creating the part to get its manifest path is cheating, a little
        info := &snap.Info{
                Name:      "foo",
                Developer: "bar",
                Version:   "1",
                Type:      pkgType,
        }

        content := `
name: foo
origin: bar
version: 1
revision: 999999999999
type: app
description: |-
  bla bla bla
iconurl: http://i.stack.imgur.com/i8q1U.jpg
downloadsize: 5554242
`
        p := snappy.RemoteManifestPath(info)
        c.Assert(os.MkdirAll(filepath.Dir(p), 0755), check.IsNil)
        c.Assert(ioutil.WriteFile(p, []byte(content), 0644), check.IsNil)

}
```

**Comment by pedronis** [comment]
merged #757 instead, this can be closed I think

**Pull request closed without merging**